### PR TITLE
docs: mec15xxevb assy6853 updated set up manual

### PR DESCRIPTION
**Pull request by maksimmasalski**
Set up manual was modified, added more detailed description how to
program board and how to set up it. Added more photos.

Signed-off-by: Maksim Masalski <maksim.masalski@intel.com>

### CODE REVIEW

**Comment by maksimmasalski** [comment]
@MaureenHelm @dbkinder Hello, I can't understand why it gives error with images. For example error is for image that was already was in that tree before I made that commit. Is it should be a jpg?

**Review by albertofloyd — APPROVED** [review]

**Review by dbkinder — CHANGES_REQUESTED** [review]

File: boards/arm/mec15xxevb_assy6853/doc/index.rst [hunk 1]
```diff
@@ -259,38 +266,89 @@ Building
...
+
+    .. image:: ./spidongle_assy6791_view1.png
+         :width: 400px
+         :align: center
+         :alt: SPI DONGLE ASSY 6791 Connected
+
+#. Connect programmer to the header J6 on the Assy6791 board, it will flash the SPI NOR chip ``U3``
+   Make sure that your programmer's offset is 0x0.
+   For programming you can use Dediprog SF100 or a similar tool for flashing SPI chips.
+   ..list-table:: Microchip board wiring
```
[the remark this answers is not in the document]
Comment (maksimmasalski) [reply boards/arm/mec15xxevb_assy6853/doc/index.rst]: What does the |p| means? It gives an error when I use a restview.

[the remark this answers is not in the document]
Comment (maksimmasalski) [reply boards/arm/mec15xxevb_assy6853/doc/index.rst]: If necessary to insert blank line symbol `|` works well

[the remark this answers is not in the document]
Comment (dbkinder) [reply boards/arm/mec15xxevb_assy6853/doc/index.rst]: We've got a ``substitutions.txt`` file defined in the sphinx build to set up a collection of shorthand for character replacements and stuff like that.  Using a ``|`` is OK too.

**Review by dbkinder — APPROVED** [review]
I think your doc changes look OK, but the CI system hasn't rerun the doc build so the previous error is cleared.  You may need to force push your change to give it a kick.

**Comment by maksimmasalski** [comment]
> I think your doc changes look OK, but the CI system hasn't rerun the doc build so the previous error is cleared. You may need to force push your change to give it a kick.

I ran it, it gives the same errors.

File: boards/arm/mec15xxevb_assy6853/doc/index.rst [hunk 2]
```diff
@@ -259,38 +266,96 @@ Building
...
+        :alt: SPI DONGLE ASSY 6791 Connected
+
+#. Connect programmer to the header J6 on the Assy6791 board, it will flash the SPI NOR chip ``U3``
+   Make sure that your programmer's offset is 0x0.
+   For programming you can use Dediprog SF100 or a similar tool for flashing SPI chips.
+
+   .. list-table:: Microchip board wiring
+      :align: center
+
+      * - .. image:: spidongle_assy6791.png
```
Comment (dbkinder) [inline boards/arm/mec15xxevb_assy6853/doc/index.rst]: OK, I've verified that this works (the ``.. image`` directive can only have white-space before it to work right):
```
      * -
          .. image:: spidongle_assy6791.png
             :width: 300px
             :align: center
             :alt: SPI DONGLE ASSY 6791

        -
          .. image:: spidongle_assy6791_view2.png
             :width: 300px
             :align: center
             :alt: SPI DONGLE ASSY 6791 view 2
```

BTW, PR #21509 will fix the doc processing you've encountered :)

**Review by dbkinder — CHANGES_REQUESTED** [review]
I've verified locally that these three tweaks will remove the doc build errors

**Review by dbkinder — APPROVED** [review]
+1 for doc changes, thanks.

**Pull request merged**